OpenRA.Mods.RA/Traits/GpsWatcher.cs

```diff
@@ -34,34 +34,33 @@ class GpsWatcher : ISync, IFogVisibilityModifier
 
 		readonly Player owner;
 
-		readonly List<Actor> actors = new List<Actor>();
+		readonly List<Actor> sources = new List<Actor>();
 		readonly HashSet<TraitPair<IOnGpsRefreshed>> notifyOnRefresh = new HashSet<TraitPair<IOnGpsRefreshed>>();
 
 		public GpsWatcher(Player owner)
 		{
 			this.owner = owner;
 		}
 
-		public void GpsRemove(Actor atek)
+		public void RemoveSource(Actor source)
 		{
-			actors.Remove(atek);
-			RefreshGps(atek);
+			sources.Remove(source);
+			RefreshGps(source);
 		}
 
-		public void GpsAdd(Actor atek)
+		public void AddSource(Actor source)
 		{
-			actors.Add(atek);
-			RefreshGps(atek);
+			sources.Add(source);
+			RefreshGps(source);
 		}
 
 		public void Launch(Actor atek, GpsPowerInfo info)
 		{
-			atek.World.Add(new DelayedAction(info.RevealDelay * 25,
-				() =>
-				{
-					Launched = true;
-					RefreshGps(atek);
-				}));
+			atek.World.Add(new DelayedAction(info.RevealDelay * 25, () =>
```

Comment:
I wanted to limit the scope of this pr to things that didn't require upgrade rules.